_data/events_gdcr2018/Boston.json

```diff
@@ -0,0 +1,18 @@
+{
+  "title": "Global Day of Coderetreat 2018 - Boston",
+  "url": "https://www.eventbrite.com/e/global-day-of-coderetreat-2018-boston-tickets-50447538947",
+  "moderators": [
+    "@mattc_baker",
+    "Ethan Strominger"
+  ],
+  "location": {
+    "city": "Boston",
+    "country": "United States",
```

Comment:
As of 600bb4457b63670769759639333f13120326277e there is a standardized format for US that should be used. Let's comply with that.